person.rb

```diff
@@ -4,14 +4,15 @@
 require_relative 'capitalize_decorator'
 # Class for person
 class Person
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
```

Comment:
- [ ] Please make sure to add a method in the Person class to add rentals.
  - You can consider using this code.
  
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end